README.md

```diff
@@ -38,45 +38,56 @@ The library also adds backported versions of new collection types, currently `sc
 
 ## Migration Tool
 
-We created two migration rules: 
-
-* `Collection213Upgrade` For upgrading applications (like web server, etc) from 2.11/2.12 to 2.13
-* `Collection213CrossCompat` For library that wants to cross compile to 2.11, 2.12 and 2.13
+The migration rules use scalafix. Please see the [official installation instruction](https://scalacenter.github.io/scalafix/docs/users/installation.html) and, in particular, check that your full Scala version is supported (ex 2.12.6).
 
 ```scala
 // project/plugins.sbt
 
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.7.0-RC1")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.0-RC1")
 ```
 
+### Collection213Upgrade
+
+The `Collection213Upgrade` rewrite upgrades to the 2.13 collections without the ability to compile the code-base with 2.12 or later. This rewrite is suitable for applications that don't need to cross-compile against multiple Scala versions.
```

Comment:
> without the ability to compile the code-base with 2.12 **or later**

The “or later” part is wrong.

Reply:
@MasseGuillaume Do you plan to address this comment?

Reply:
Fixed in #156